config/application.yml.default

```diff
@@ -325,7 +326,6 @@ get_usps_proofing_results_job_reprocess_delay_minutes: 5
 get_usps_proofing_results_job_request_delay_milliseconds: 1000
 voice_otp_pause_time: '0.5s'
 voice_otp_speech_rate: 'slow'
-voip_check: true
```

Comment:
I expect we're using the default value in most live environments, but it might be a good idea to do a pass for how it's configured, and update for the rename or remove redundant configurations.